section3/exercises/loops-arrays.js

```diff
@@ -18,22 +18,35 @@ for (var i = 0; i < animals.length; i++) {
 
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
+var canidaes = ["Coyotes","Wolves","Foxes"];
 
-
+for (var i = 0; i < canidaes.length; i++) {
+  console.log(canidaes[i] + " are awesome!");
+}
 
 // YOU DO: Create an array of foods and then iterate over that array to log "Add
 // <food> to shopping list" for each food item.
+var foods = ["avacados","blueberries","bananas","yogurt","rice noodles"];
 
+for (var i = 0; i < foods.length; i++) {
+  console.log("Add "+foods[i]+ "to shopping list");
+}
 
 
 // YOU DO: Create an array of numbers and then iterate over that array to log double
 // of each of the number. (Ex: 4 doubled is 8.)
+var numbers = [6,11,3,21,32];
 
-
+for (var i = 0; i < numbers.length; i++) {
+  console.log(numbers[i] * 2);
+}
 // YOU DO: Using the array of names below, print out "Happy Birthday <name>" for
 // each name except the first two.
 var names = ["Sevy", "Lindsey", "Jericho", "Raeonna", "Latrina", "Jhana", "Luca"];
 
+for (var i = 2; i < names.length; i++) {
+  console.log("Happy Birthday "+names[i]+"!");
+}
```

Comment:
I'd challenge you to favor string interpolation over concatenation going forward—much easier for other developers to read when your strings start to get longer/include more variables 🙂